From 8f90484f7c5bf273adc85596746cee57a63dd9f6 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Fri, 24 Dec 2021 21:55:56 +0100
Subject: [PATCH] Fix building on aarch64

---
 src/common/darktable.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/common/darktable.h b/src/common/darktable.h
index b841bf2bc15..aad6169f8b6 100644
--- a/src/common/darktable.h
+++ b/src/common/darktable.h
@@ -133,6 +133,8 @@ typedef unsigned int u_int;
 # elif defined(__PPC64__)
 /* __PPC64__ is the only macro tested for in is_supported_platform.h, other macros would fail there anyway. */
 #define __DT_CLONE_TARGETS__ __attribute__((target_clones("default","cpu=power9")))
+# else
+#define __DT_CLONE_TARGETS__
 # endif
 #else
 #define __DT_CLONE_TARGETS__
