diff -up darktable-3.8.0/src/common/dttypes.h.omv~ darktable-3.8.0/src/common/dttypes.h
diff -up darktable-3.8.0/src/develop/noise_generator.h.omv~ darktable-3.8.0/src/develop/noise_generator.h
--- darktable-3.8.0/src/develop/noise_generator.h.omv~	2021-12-31 03:17:18.977165411 +0100
+++ darktable-3.8.0/src/develop/noise_generator.h	2021-12-31 03:21:07.366978907 +0100
@@ -211,8 +211,7 @@ static inline void poisson_noise_simd(co
   // now we have gaussian noise, then apply Anscombe transform to get poissonian one
   dt_aligned_pixel_t r = { 0.f };
 
-  #pragma unroll
-  for_each_channel(c)
+  for_each_channel(c, unroll)
   {
     r[c] = noise[c] * sigma[c] + 2.0f * sqrtf(fmaxf(mu[c] + 3.f / 8.f, 0.0f));
     out[c] = (r[c] * r[c] - sigma[c] * sigma[c]) / 4.f - 3.f / 8.f;
