diff --git a/src/develop/noise_generator.h b/src/develop/noise_generator.h
index b3451f1..ebb0c38 100644
--- a/src/develop/noise_generator.h
+++ b/src/develop/noise_generator.h
@@ -221,8 +221,7 @@ static inline void poisson_noise_simd(const dt_aligned_pixel_t mu,
   // Now we have gaussian noise, then apply Anscombe transform to get poissonian one.
   dt_aligned_pixel_t r = { 0.f };
 
-  #pragma unroll
-  for_each_channel(c)
+  for_each_channel(c, unroll)
   {
     r[c] = noise[c] * sigma[c] + 2.0f * sqrtf(fmaxf(mu[c] + 3.f / 8.f, 0.0f));
     out[c] = (r[c] * r[c] - sigma[c] * sigma[c]) / 4.f - 3.f / 8.f;
