diff -uraN darktable-4.6.1/CMakeLists.txt omv-darktable-4.6.1/CMakeLists.txt
--- darktable-4.6.1/CMakeLists.txt	2024-02-16 18:31:09.000000000 +0100
+++ omv-darktable-4.6.1/CMakeLists.txt	2024-04-30 20:38:50.534686312 +0200
@@ -81,7 +81,7 @@
 
         # Clang has an option to specify the OpenMP standard to use. Specify it.
         # FIXME: Implement this in FindOpenMP.cmake
-        set(OPENMP_VERSION_SPECIFIER "-fopenmp-version=45")
+        set(OPENMP_VERSION_SPECIFIER "-fopenmp-version=50")
 
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OPENMP_VERSION_SPECIFIER}")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPENMP_VERSION_SPECIFIER}")
diff -uraN darktable-4.6.1/src/external/rawspeed/cmake/src-dependencies.cmake omv-darktable-4.6.1/src/external/rawspeed/cmake/src-dependencies.cmake
--- darktable-4.6.1/src/external/rawspeed/cmake/src-dependencies.cmake	2024-02-14 23:08:05.000000000 +0100
+++ omv-darktable-4.6.1/src/external/rawspeed/cmake/src-dependencies.cmake	2024-04-30 20:40:29.394049100 +0200
@@ -39,7 +39,7 @@
 
     # NOTE: we want at least 5.0, but we don't need full implementation,
     # so we neither can't really check for a version, not need to...
-    find_package(OpenMP)
+    find_package(OpenMP 4.5)
 
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS_SAVE}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS_SAVE}")
