diff -uraN darktable-4.6.0/src/develop/noise_generator.h omv-darktable-4.6.0/src/develop/noise_generator.h
--- darktable-4.6.0/src/develop/noise_generator.h	2023-12-14 20:44:41.000000000 +0100
+++ omv-darktable-4.6.0/src/develop/noise_generator.h	2023-12-24 15:04:27.817312453 +0100
@@ -241,8 +241,7 @@
   // Now we have gaussian noise, then apply Anscombe transform to get poissonian one.
   dt_aligned_pixel_t r = { 0.f };
 
-  #pragma unroll
-  for_each_channel(c)
+  for_each_channel(c, unroll)
   {
     r[c] = noise[c] * sigma[c] + 2.0f * sqrtf(fmaxf(mu[c] + 3.f / 8.f, 0.0f));
     out[c] = (r[c] * r[c] - sigma[c] * sigma[c]) / 4.f - 3.f / 8.f;
