diff -uraN darktable-4.2.0/src/is_supported_platform.h omv-darktable-4.2.0/src/is_supported_platform.h
--- darktable-4.2.0/src/is_supported_platform.h	2022-12-14 16:18:18.000000000 +0100
+++ omv-darktable-4.2.0/src/is_supported_platform.h	2022-12-24 19:40:37.151786245 +0100
@@ -30,7 +30,7 @@
 #define DT_SUPPORTED_X86 0
 #endif
 
-#if defined(__aarch64__) && (defined(__ARM_64BIT_STATE) && defined(__ARM_ARCH) && (defined(__ARM_ARCH_8A) || __ARM_ARCH_PROFILE == 'A') || defined(__APPLE__) || defined(__MINGW64__))
+#if defined(__aarch64__)
 #define DT_SUPPORTED_ARMv8A 1
 #else
 #define DT_SUPPORTED_ARMv8A 0
